### PR TITLE
Add icon and state formatting for tts and stt domain

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

CleanShot 2023-04-25 at 18 18 51

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by piitaya** [comment]
We can keep the current TTS icon but we need to find one better for STT.
May be just a microphone ?

CleanShot 2023-04-25 at 18 00 58

**Comment by piitaya** [comment]
or wave for STT
CleanShot 2023-04-25 at 18 04 28

**Comment by bramkragten** [comment]
I kind agree with you, but we have to align with brand, media player, etc.

Btw, `conversation` is also using `mdiMicrophoneMessage`...

**Comment by piitaya** [comment]
Only brand should be updated right? What do we have to update to media player?
For conversation, I don't think it's used in the frontend.

**Comment by bramkragten** [comment]
> Only brand should be updated right? What do we have to update to media player? For conversation, I don't think it's used in the frontend.

yeah media panel uses brands

File: src/common/entity/domain_icon.ts [hunk 1]
```diff
@@ -232,6 +234,9 @@ export const domainIconWithoutDefault = (
       break;
     }
 
+    case "stt":
+      return mdiMicrophoneMessage;
```
Comment (bramkragten) [inline src/common/entity/domain_icon.ts:238]: Since these are static, they should be added to `FIXED_DOMAIN_ICONS`

**Review by frenck — APPROVED** [review]
LGTM

**Pull request merged**